cli/util/git_test.go

```diff
@@ -53,13 +44,46 @@ func TestGetRepoName(t *testing.T) {
 			repo:   "https://github.com/foo/bar",
 			subDir: "modules/bp1",
 			remote: defaultRemote,
-			want:   "bar",
 		},
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			dir := tempGitRepoWithRemote(t, tt.repo, tt.remote, tt.subDir)
-			got, err := GetRepoName(dir)
+			_, err := GetRepoUrl(dir)
```

Comment:
Shouldn't we be checking the return from GetRepoUrl to verify it equals tt.repo?